src/components/v2/appDetails/sourceInfo/environmentStatus/EnvironmentStatus.component.tsx

```diff
@@ -153,7 +153,11 @@ function EnvironmentStatusComponent({
         return (
             appDetails?.lastDeployedTime && (
               <DeploymentStatusCard
+              isGitops={isGitops}
               deploymentStatusDetailsBreakdownData={deploymentStatusDetailsBreakdownData}
+              hideDeploymentStatusLeftInfo={!isGitops}
```

Comment:
can't we use isGitops directly as it is available on line 156

Reply:
Removed unnecessary one